addons/website_blog/static/src/js/blog_tag_add.js

```diff
@@ -0,0 +1,320 @@
+odoo.define('website_blog.blog_post_tag.add', function (require) {
+'use strict';
+
+const {qweb, _t} = require('web.core');
+const options = require('web_editor.snippets.options');
+const {Class: EditorMenuBar} = require('web_editor.editor');
+
+options.registry.TagBlogPostSelect = options.Class.extend({
+    xmlDependencies: (options.Class.prototype.xmlDependencies || []).concat(['/website_blog/static/src/xml/website_blog_tag.xml']),
+    /**
+     * @override
+     */
+    willStart: async function () {
+        const _super = this._super.bind(this);
+        this.blogPostId = this.$target.data('blog-id');
+        this.addList = [];
+        this.unlinkList = [];
+        this.isEditing = false;
+        this.tagIds = [];
+        this.availableTags = [];
+        this.trigger_up('set_blog_post', {
+            blogPostId: this.blogPostId,
+        });
```

Comment:
Why notifying the blog post id now? Could not it be notified alongside the new tags so there is only one trigger_up event? Also it would naturally allow to edit tags from multiple blog post from the "all blog posts" page.